### DISCUSSION TITLE
Custom label in New Label Engine

### DISCUSSION THREAD

**Discussion by follet-git** [snipe/snipe-it#13949] [Q&A]
Hello, I use Snipe-IT Version v6.2.3.
I like the new feature but I see only 9 label sizes.
I use a Godex G500 Label printer with labels size: 40 x 20 mm.

- I would like to create custom preset #10 with my label size
- Also, I need to know the "DataSource" values to insert data to label
Can you help me with it?

**Comment by MaKaNu** [comment]
I am at the same position. What is the current process for this?

For example we have:

- Brother DK-22212
- Brother DK-11209

Which are not in the current version available.

Is the current intended way (in a docker scenario) to add a Volume under Tapes/Custom?
